### PR TITLE
Made an installation script.

### PR DESCRIPTION
**Pull request by Greenscreener**

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,98 @@
+#/bin/bash
```
Comment (vojta001) [inline install.sh]: Shebang starts with #! - `#!/bin/bash`

**Comment by Greenscreener** [comment]
The shebang should now be fixed.

**Comment by vojta001** [comment]
I really appreciate your effort, but this does not satisfy my expectations of "installation script". I was thinking about the typical `./configure`, `make` and `make install` or a distribution specific package (Archlinux's one is AFAIK easy to make).

**Comment by vojta001** [comment]
Of course if you have any other ideas, I am looking forward to them. (or fork this and keep them anyway. OSS world is awesome)

The problem with your previous one is, it only installs Python and Git. Both of them are usually included in a typical Linux distribution, so the only effect was running `git clone`, which is just as difficult as running the script.

**Pull request closed without merging**